src/dotnet/CommonLocalizableStrings.resx

```diff
@@ -628,4 +628,13 @@ setx PATH "%PATH%;{0}"
   <data name="ToolSettingsInvalidLeadingDotCommandName" xml:space="preserve">
     <value>The command name '{0}' cannot begin with a leading dot (.).</value>
   </data>
-</root>
+  <data name="FormatVersionIsHigher" xml:space="preserve">
+    <value>Format version is higher than supported. This tool may not be supported in this SDK version. Please update SDK.</value>
```

Comment:
suggestion:`Format version is higher than supported. This tool may not be supported in this SDK version. Please update your SDK`. (added *your*)